### ISSUE TITLE
Separating clang and share the same cindex with other clang plugins

### ISSUE THREAD

**Issue by arakashic** [arakashic/chromatica.nvim#9]
Need to investigate the interoperability between plugins

**Comment by romix** [comment]
This can be related to this issue:
https://github.com/davits/DyeVim

**Comment by arakashic** [comment]
Yes. This is related to what I have been thinking about. Although I would prefer depending on something lighter than YCM. :-)
